### ISSUE TITLE
[Bug]: v0.20.0 netclient version is still reported as being v0.19 on web gui after netclient upgrade to v0.20.0

### ISSUE THREAD

**Issue by jalbstmeijer** [gravitl/netmaker#2323]
### What happened?

Upgraded both netmaker and netclient docker images to v0.20.0 .
The admin web gui still reports v0.19 for the netclient hosts.

### Version

v0.20.0

### Relevant log output

```shell
[netclient] 2023-05-20 09:51:56 netclient daemon started -- version: v0.20.0
```
```


### Contributing guidelines

- [X] Yes, I did.

**Comment by mattkasun** [comment]
planned for v0.20.1

**Comment by mattkasun** [comment]
fixed in v0.20.1
